public/app/features/templating/formatRegistry.ts

```diff
@@ -16,10 +16,29 @@ export interface FormatRegistryItem extends RegistryItem {
   formatter(options: FormatOptions, variable: VariableModel): string;
 }
 
+export enum FormatRegistryID {
+  lucene = 'lucene',
+  raw = 'raw',
+  regex = 'regex',
+  pipe = 'pipe',
+  distributed = 'distributed',
+  csv = 'csv',
+  html = 'html',
+  json = 'json',
+  percentEncode = 'percentencode',
+  singleQuote = 'singlequote',
+  doubleQuote = 'doublequote',
+  sqlString = 'sqlstring',
+  date = 'date',
+  glob = 'glob',
+  text = 'text',
+  queryParam = 'queryparam',
```

Comment:
i meant like 
```typescript
  Json = 'json',
  PercentEncode = 'percentencode',
  SingleQuote = 'singlequote',
  ...etc
```
but tbh looking at the rest of the codebase we're not super consistent on this 🤷

Reply:
Aha ok, I was looking at the IDs in `packages/grafana-data/src/transformations/matchers/ids.ts`